lib/resources/r4/condition.yaml

```diff
@@ -263,6 +268,26 @@ fields:
   note: subject must be a reference to a Patient.
   url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.subject
 
+- name: encounter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference
+  description: Encounter during which the condition was first asserted.
+  example: |
+    {
+      "encounter": {
+        "reference": "Encounter/631923"
+      }
+    }
+  note: |
+    <ul>
+      <li> encounter must be a reference to an Encounter. </li>
+      <li> encounter during which the condition was first asserted. </li>
```

Comment:
Might not need this here since it's already in the description.

Should we mention that the encounter can't actually be updated?

Reply:
Removed: https://github.com/cerner/fhir.cerner.com/pull/365/commits/ad905dea3880b346653ab91e60519e03aecd5bbf

They don't mention it in dstu2 and I was going off of that. https://fhir.cerner.com/millennium/dstu2/general-clinical/condition/#update